packages/utils/src/commonwell/migrate-patients.ts

```diff
@@ -52,6 +50,7 @@ const PATIENT_CHUNK_SIZE = 2;
 
 const oldCwOid = "";
 const cwOrgName = "";
+const orgNpi = "";
```

Comment:
Can you add a comment about the expected format here (with or without `urn:oid`), as well as update the script's TSDoc to account for these vars not coming from env vars but having to be populated here please?